File: src/replay_test/replay_test_workflow.erl
-module(replay_test_workflow).

% elp:ignore W0012 W0040
-moduledoc false.

-export([
    execute/2
]).

-include_lib("temporal_sdk/include/workflow.hrl").

execute(_Context, _Input) ->
    start_timer(1),
    start_activity(echo_activity, [[~"hello world"]]),
    record_uuid4().

File: lib/replay_test/replay_test_workflow.ex
defmodule ReplayTest.Workflow do
  use TemporalSdk.Workflow
  @moduledoc false

  @impl true
  def execute(_context, _input) do
    start_timer(1)
    start_activity(:echo_activity, [["hello world"]])
    record_uuid4()
  end
end
